src/chat/chatter.py

```diff
@@ -262,7 +258,9 @@ def updatePlayer(self):
             self.setIcon(util.THEME.icon("chat/countries/%s.png" % country.lower()))
             self.setToolTip(country)
 
-        self.updateAvatar()
+        if player.avatar != self.avatar:
```

Comment:
There's no avatar member set in __init__, it probably didn't survive the rebase. It doesn't affect sorting anyway, so we can keep it how it is.